kafka-connect/src/main/java/io/tabular/iceberg/connect/channel/Coordinator.java

```diff
@@ -168,6 +186,15 @@ private void doCommit(boolean partialCommit) {
         vtts);
   }
 
+  private String resolveOffsetsJson(List<Envelope> envelopeList, Map<Integer, Long> oldOffset) {
```

Comment:
Better to set the type as `LinkedList` upfront since we only support LinkedList?
```suggestion
  private String resolveOffsetsJson(LinkedList<Envelope> envelopeList, Map<Integer, Long> oldOffset) {
```

Reply:
Actually after a thought I am refactoring to not use LinkedList method as [this](https://github.com/anmol/iceberg-kafka-connect/pull/1/files#diff-281073fed03b58b2f4c5bc24dad0c69fa31cf4bef2f9b4af164830bc489bd2b3R154) changes the linkedList to an immutable list.